Libraries/LibWeb/HTML/HTMLImageElement.cpp

```diff
@@ -412,13 +412,13 @@ Optional<ARIA::Role> HTMLImageElement::default_role() const
     // https://www.w3.org/TR/html-aria/#el-img
     // https://www.w3.org/TR/html-aria/#el-img-no-alt
     // https://w3c.github.io/aria/#image
-    // N.B. The “image” role value is a synonym for the older “img” role value; however, the el-img test in
-    // https://wpt.fyi/results/html-aam/roles.html expects the value to be “image” (not “img”).
+    // Note: The "image" role value is a synonym for the older "img" role value; however, the el-img test in
```

Comment:
This was squashed into the wrong commit. (and we use NOTE, all capitals).

Reply:
oops, thanks — fixed now